code/modules/supermatter/supermatter.dm

```diff
@@ -84,12 +84,12 @@ var/global/list/supermatter_final_thoughts = list(
 /obj/machinery/power/supermatter
 	name = "supermatter crystal"
 	desc = "A strangely translucent and iridescent crystal. <span class='danger'>You get headaches just from looking at it.</span>"
-	icon = 'icons/obj/engine.dmi'
-	icon_state = "darkmatter"
+	icon = 'icons/obj/supermatter.dmi'
+	icon_state = "supermatter"
 	density = 1
 	anchored = 0
 	light_range = 4
-	layer = ABOVE_OBJ_LAYER
+	layer = ABOVE_HUMAN_LAYER
```

Comment:
I mean, I still hate this. Would much rather have the sprite be the correct size instead.

Reply:
My main issue with this comment is that the complaint is based around a false pretense, there's other objects in the game which are larger than 32x32. They are just also set on the incorrect `ABOVE_OBJ_LAYER` so they phase above Human mobs, so you just didn't notice them searching the code.

Reply:
It's not a false pretense to say that he dislikes it and would prefer if they were 32x32. There are not many larger icons in the game and the ones that are in general circulation tend to either be problematic (multitile airlocks) or inaccessible (RUST cores).

Reply:
I believe in a similar comment on the earlier PR, I said other objs don't do this: I stand corrected, but my overall sentiment remains unchanged.